recipes/openmm-xtb/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "openmm-xtb" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-xtb/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 94778d570977f2f67df9330775c01b749bf6323cae76120067e39fc8601ed9f6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - swig                                   # [build_platform != target_platform]
+    - openmm >=8.1.1                         # [build_platform != target_platform]
+    - cmake
+    - make
+    - conda-build  # [osx]
+  host:
+    - python
+    - pip
+    - openmm >=8.1.1
+    - xtb
+    - swig                                   # [not arm64]
+  run:
+    - python
+    - {{ pin_compatible('openmm', min_pin='x.x', max_pin='x.x') }}
```

Comment:
Aren't the libraries using a run_exports? Normally you shouldn't need to do this for native libraries.

Reply:
Apparently not.  There's no `run_exports` in https://github.com/conda-forge/openmm-feedstock/blob/main/recipe/meta.yaml.